DocumentDataAPI/DocumentDataAPITests/Data/Repositories/NpgSourceRepositoryIntegrationTests.cs

```diff
@@ -37,94 +37,94 @@ public void GetAll_ReturnsAllSources()
     }
 
     [Fact]
-    public void Update_UpdatesRow()
+    public async void Update_UpdatesRow()
     {
         // Arrange
         NpgSourceRepository repository = new(_connectionFactory, _logger);
         const string expected = "Test Source";
 
         // Act
-        repository.Update(new SourceModel(1, expected));
-        string? actual = repository.Get(1)?.Name;
+        await repository.Update(new SourceModel(1, expected));
+        string? actual = (await repository.Get(1))?.Name;
 
         // Assert
         actual.Should().NotBeNull()
             .And.Be(expected, "because the tuple (1, 'DR') was updated to (1, 'Test Source')");
     }
 
     [Fact]
-    public void DeleteWithNoForeignKeyViolation_DeletesRow()
+    public async void DeleteWithNoForeignKeyViolation_DeletesRow()
     {
         // Arrange
         NpgSourceRepository repository = new(_connectionFactory, _logger);
         IEnumerable<SourceModel> expected = new List<SourceModel>() { new(1, "DR"), new(2, "TV2") };
         SourceModel newSource = new(3, "Test");
-        repository.Add(newSource);
+        await repository.Add(newSource);
 
         // Act
-        repository.Delete(newSource);
-        IEnumerable<SourceModel> actual = repository.GetAll();
+        await repository.Delete(newSource);
+        IEnumerable<SourceModel> actual = await repository.GetAll();
 
         // Assert
         actual.Should().BeEquivalentTo(expected);
     }
 
     [Fact]
-    public void Add_AddsNewSource()
+    public async void Add_AddsNewSource()
```

Comment:
I started this, _and I am going to finish this!_
```suggestion
    public async Task Add_AddsNewSource()
```